src/components/v5/payments/TmpAdvancedPayments.tsx

```diff
@@ -98,6 +106,20 @@ const TmpAdvancedPayments = () => {
     });
   const streamingPayment = streamingPaymentData?.getStreamingPayment;
 
+  const [getStreamingPayments] = useLazyQuery(
+    gql`
+      query ListStreamingPayments {
+        listStreamingPayments {
+          nextToken
+          items {
+            nativeId
+          }
+        }
+      }
+    `,
```

Comment:
This is okay since it's a testing UI, otherwise it should live in a `.graphql` file

Reply:
My thoughts exactly, I didn't wanna add this to the `streamingPayments.graphql` file:
```gql
query ListStreamingPayments {
  listStreamingPayments {
    items {
      ...StreamingPayment
    }
  }
}
```
Because I didn't wanna pollute the `generated.ts` file with a new query, lazy query etc. that won't be used in prod, at least from the look of things!